### ISSUE TITLE
pesq and SISNR weight problem

### ISSUE THREAD

**Issue by cwhm123** [audiolabs/torch-pesq#10]
I see that using your PESQ loss and SISDR loss together can achieve good results. What is the weight ratio between the two during training

**Comment by bytesnake** [comment]
equal weighting (lambda=1.0) but to achieve good results you should to hyperparameter search
